### PR TITLE
DEVELOPER-3844 Cheat sheets minor author label fix

### PR DESCRIPTION
**Pull request by adelasofia**
Hides cheat sheet author from page if none was provided.

### CODE REVIEW

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/node--cheat-sheet.html.twig [hunk 1]
```diff
@@ -75,7 +75,9 @@ other methods (such as node.delete) will result in an exception.
         <div class="cs-info">
             <ul>
                 <li>
-                    <strong{{ author_attributes }}>By {{content.field_cheat_sheet_author}} </strong>
+                    {% if content.field_cheat_sheet_author|render is not empty %}
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/node--cheat-sheet.html.twig]: Typically we've also added strip_tags|trim as well as the render. That will guarantee you either have an empty string or not.

Comment (adelasofia) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/node--cheat-sheet.html.twig]: Just pushed the change 👍

**Comment by adelasofia** [comment]
retest this please

**Comment by adelasofia** [comment]
retest this please

**Comment by adelasofia** [comment]
retest this please

**Pull request closed without merging**